homeassistant/components/unifi/device_tracker.py

```diff
@@ -154,10 +155,14 @@ def __init__(self, client, controller):
             if self._is_connected:
                 self.schedule_update = True
 
+    async def async_added_to_hass(self) -> None:
+        """Watch object when added."""
+        self.controller.add_disconnected_check(self)
```

Comment:
Entities should not be stored in a shared container.

Reply:
Addressed in https://github.com/home-assistant/core/pull/45211